.storybook/components/ElementNamer.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+
+export const TableNamer = ({ names }) => {
+  useEffect(() => {
+    const tables = document.querySelectorAll('table:not([aria-hidden="true"])');
+    if (tables.length !== names.length) {
+      console.error(
+        '🦒: TableNamer: number of tables does not match number of names',
+        `Found ${tables?.length ?? 0} and was provided ${names?.length ?? 0}`
+      );
+    }
+    for (let i = 0; i < tables.length; i++) {
+      tables[i].setAttribute('title', names[i]);
+    }
+  }, []);
+  return <></>;
+};
+
+export const CopyButtonNamer = ({ names }) => {
+  useEffect(() => {
+    window.addEventListener('load', () => {
```

Comment:
Why is this registering an event listener for window load? 
Given that this is after the code blocks in dom order the HTML that this is intended to mutate should have already been rendered.

Reply:
I had an issue with getElementsByClassName returning a live collection with the length of the object 0, which I discovered was because at that point of time there is no element with that className in the DOM. Listening for the load event fixed this.